src/draw/lv_draw_image.c

```diff
@@ -116,16 +114,52 @@ void lv_draw_image(lv_layer_t * layer, const lv_draw_image_dsc_t * dsc, const lv
         return;
```

Comment:
There is a `LV_PROFILER_DRAW_END` here. In addition, you need to add `LV_PROFILER` to the `custom_draw` process to facilitate the detection of performance issues.

Reply:
done